dtmi/azureiot/phone_as_a_device/sensors/accelerometer-2.json

```diff
@@ -0,0 +1,27 @@
+{
+    "@context": [
+      "dtmi:dtdl:context;2",
+      "dtmi:iotcentral:context;2"
+    ],
+    "@id": "dtmi:azureiot:Phone_As_A_Device:Sensors:Accelerometer;2",
+    "@type": "Interface",
+    "extends": ["dtmi:azureiot:Phone_As_A_Device:Sensors:baseSensor;1"],
```

Comment:
AFAIK, Central does not support extends in components

Reply:
The -2 versions of sensors are DTDL v3 compliant (with extends). The versions in use in the app are all the -1 sensors which are flattened and do not extend the base sensor.